### PR TITLE
fix(client): Design Misalignment in Project Deletion UI

### PR DESCRIPTION
**Pull request by TheShiveshNetwork**
Close: #7276

## PR Details

This PR redesigns the project deletion UI as mentioned in the Issue.

### Summary
🎨📝🚧

Add custom styles to `Panel` component in `ProjectFormPage.tsx`. This is a demo of how to use the `style` prop to change the look of the panel.

> _`Panel` gets new style_
> _Red border, pink background_
> _Just for demo - kireji - spring_

### Walkthrough
*  Add a red border and a pink background to the project name and description panel ([link](https://github.com/amplication/amplication/pull/7323/files?diff=unified&w=0#diff-02e6a440827a1ae5bc4371de6c9e0223b694b696995593041432cdc3b267c8b6L25-R30))
*  Use the `style` prop of the `Panel` component to customize its appearance ([link](https://github.com/amplication/amplication/pull/7323/files?diff=unified&w=0#diff-02e6a440827a1ae5bc4371de6c9e0223b694b696995593041432cdc3b267c8b6L25-R30))

## PR Checklist

- [x] Tests for the changes have been added
- [x] `npm test` doesn't throw any error

IMPORTANT: Please review the [CONTRIBUTING.md](https://github.com/amplication/amplication/blob/master/CODE_OF_CONDUCT.md) file for detailed contributing guidelines.

### CODE REVIEW

File: packages/amplication-client/src/Project/ProjectFormPage.tsx [hunk 1]
```diff
@@ -22,7 +22,12 @@ function ProjectFormPage() {
         <Text textStyle={EnumTextStyle.H4}>Delete Project</Text>
       </FlexItem>
 
-      <Panel>
+      <Panel
```
Comment (barshimi) [inline packages/amplication-client/src/Project/ProjectFormPage.tsx:25]: why not using class instead of style property

**Comment by TheShiveshNetwork** [comment]
@barshimi , if this doesn't have any issue, can you please approve this?

**Comment by mulygottlieb** [comment]
@TheShiveshNetwork - you should follow the instructions from @barshimi about not adding the style hard-coded in this way into the `.tsx` file. It's not aligned with our coding guidelines.
I'll give you another week to amend your PR if you wish to get it merged (and included as part of our Hacktoberfast since you submitted it on time)

**Review by TheShiveshNetwork — COMMENTED** [review]
Please go through the latest commit.

**Comment by TheShiveshNetwork** [comment]
@mulygottlieb and @barshimi Do tell me, if this is fine.

File: packages/amplication-client/src/Project/ProjectPage.scss [hunk 2]
```diff
@@ -1,3 +1,8 @@
 @import "../style/index.scss";
 .project-page {
 }
+
+.delete__panel {
+    border: 1px solid red;
+    background: rgba(255, 20, 70, 0.4);
```
Comment (yuval-hazaz) [inline packages/amplication-client/src/Project/ProjectPage.scss:7]:
```suggestion
    background: var(--theme-light-red-transparent)
```

File: packages/amplication-client/src/Project/ProjectPage.scss [hunk 3]
```diff
@@ -1,3 +1,8 @@
 @import "../style/index.scss";
 .project-page {
 }
+
+.delete__panel {
+    border: 1px solid red;
```
Comment (yuval-hazaz) [inline packages/amplication-client/src/Project/ProjectPage.scss:6]:
```suggestion
    border: 1px solid var(--theme-red);
```

File: packages/amplication-client/src/Project/ProjectFormPage.tsx [hunk 4]
```diff
@@ -22,7 +23,9 @@ function ProjectFormPage() {
...
+        className={`delete__panel`}
+      >
         <FlexItem itemsAlign={EnumItemsAlign.Center}>
           <FlexItem.FlexStart>
             <Text textStyle={EnumTextStyle.Description}>
```
Comment (yuval-hazaz) [inline packages/amplication-client/src/Project/ProjectFormPage.tsx:31]:
```suggestion
            <Text textColor={enumTextColor.White} textStyle={EnumTextStyle.Description}>
```

**Review by yuval-hazaz — CHANGES_REQUESTED** [review]
thanks for your contribution and sorry it took time to reply.
please see my comments, please apply the changes and share a screenshot of the new results on the description of the PR

**Review by yuval-hazaz — APPROVED** [review]

**Pull request merged**